src/common-ssh/ssh.c

```diff
@@ -360,7 +361,7 @@ static int guac_common_ssh_authenticate(guac_common_ssh_session* common_session)
 
     /* Attempt authentication with username + password. */
     if (user->password == NULL && common_session->credential_handler)
-            user->password = common_session->credential_handler(client, "Password: ");
+        common_session->credential_handler(client, GUAC_SSH_PARAMETER_NAME_PASSWORD);
```

Comment:
For SSH/telnet, this should probably support both the old-style terminal prompt and the prompt leveraging `required`, internally selecting between these options depending on owner support.

Reply:
Yep, agreed.